### ISSUE TITLE
Incorrect data model version label on RDBES web site

### ISSUE THREAD

**Issue by davidcurrie2001** [ices-tools-dev/RDBES#207]
The version label on the RDBES web site says "_The data model behind this version of the RDBES is data model version 1.19.4_" - is that correct?  I think it is using a later version e.g. v1.19.26

**Comment by HenrikK-N** [comment]
You are right the version should be updated, but we are not using the individual CL CE, VD SL or CS versions, we are using the documentation version, so it should be updated to version 1.19.5.

**Comment by rix133** [comment]
Still it is really hard to understand waht verison the data model is so it is hard to make R code compatible with a version here are the current version codes:
https://rdbes.ices.dk/home (when logged in):
>   The data model behind this version of the RDBES is data model version 1.19.4

The RDBES Data Model VD SL.xlsx
> Ver 1.20

RDBES Documentation of the Data Model.docx
>RDBES Data Model doc. v. 1.19.6

RDBES Data Model CL CE.xlsx
>Ver 1.19.34

RDBES Data Model CS.xlsx
>Model v1.19.27

Woud it be possible to harmonize the version numbering (at least for the tightly related things).  I udnerstand that you fix typos etc in the xlsx files so maybe if it is a typo fix you say minor fix ie (change the last number) but when you add/remove columns its allready a version change at the second level when you add remove tables its a change from 1>2  Then in the documentation you could omit minor version and say only if it's 1.2 or 2.0 

Just maybe ther could be a current version bump in the readme.md when any fixes are made or an other from of consistency woud be nice?

**Comment by HenrikK-N** [comment]
As wrote in the answer above "we are not using the individual CL CE, VD SL or CS versions, we are using the documentation version." The new version is 1.19.6. The is the version number of the documentation document and the version of the RDBES system on the ICES web site.
